apis/visibility/v1alpha1/types.go

```diff
@@ -84,10 +84,10 @@ type PendingWorkloadOptions struct {
 	metav1.TypeMeta `json:",inline"`
 
 	// Offset indicates position of the first pending workload that should be fetched starting from 0. 0 by default
-	Offset int32 `json:"offset"`
+	Offset int `json:"offset"`
```

Comment:
use int32 or int64, as necessary.

Reply:
int32 or int64 are not supported by the conversion framework and using them would require manual conversion

Reply:
How so? we use these types in k/k extensively.

What is the conversion for?

Reply:
> What is the conversion for?

`func autoConvert_url_Values_To_v1alpha1_PendingWorkloadOptions(in *url.Values, out *PendingWorkloadOptions, s conversion.Scope) error {` in `zz_generated.conversion.go`.

Reply:
I've double-checked, and to my surprise, int64 is supported but int32 is not. In general, the conversion is used to convert `url.Values` (query parameters) into `PendingWorkloadOptions`. 

I'll use int64 then.